frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.test.tsx

```diff
@@ -156,85 +158,122 @@ describe("StreamlitMarkdown", () => {
     )
   })
 
-  it("doesn't render invalid markdown (tables, images, etc.) when isLabel is true", () => {
-    // Invalid Markdown in widget/expander/tab labels
-    const table =
-      "<table><tr><th>Month</th><th>Savings</th></tr><tr><td>January</td><td>$100</td></tr></table>"
-    const image =
-      "![Corgi](https://dictionary.cambridge.org/us/dictionary/english/corgi)"
-    // Valid Markdown in widget/expander/tab labels
-    const valid =
-      "*Italics* ~Strikethrough~ **Bold** :traffic_light: `code` [Yikes](http://msdn.microsoft.com/en-us/library/aa752574(VS.85).aspx)"
-
-    const wrapper1 = mount(
-      <StreamlitMarkdown source={table} allowHTML={true} isLabel />
-    )
-    expect(wrapper1.find("StyledStreamlitMarkdown").text()).toEqual("")
-    expect(wrapper1.props().isLabel).toEqual(true)
+  it("renders valid markdown when isLabel is true", () => {
```

Comment:
this should likely be refactored to it.each(...)  

https://jestjs.io/docs/api#testeachtablename-fn-timeout